January 2023

baxtan opened the pull request:
The VL.Addons; It came in a few hours ago, and I included it. Please double-check.

leavittx commented:
Test comment

antongit commented:
Test comment

joreg commented on content/blog/2022/other/news_in_hap_world.md:
```diff
@@ -0,0 +1,70 @@
+author: lev
+---
+
+There were multiple important things happening in the Hap GPU-accelerated video playback codec world recently. Sadly, there is no a single website with all these updates (though there is an [official website](http://hap.video). I will try to put the information all together in this blog post.
```
_not_ a single website..

joreg commented on content/blog/2022/other/news_in_hap_world.md:
```diff
@@ -0,0 +1,70 @@
+### Hap player for vvvv updates
+Next, I'd like to list what new features were added to the vvvv's Demolition Media Hap Player:
+1. The one I especially like — texture updates are done now in a background thread — main thread is unaffected by playback
+2. The loop playback has became seamless (thanks to pre-caching next frames in a non-stop manner). No hiccups at the start of new loop whether you have custom start/end frames set or not
```
has _become_ seamless

leavittx replied:
oh, right

joreg commented on content/blog/2022/other/news_in_hap_world.md:
```diff
@@ -0,0 +1,70 @@
+
+**It's possible to send invoices on a regular basis -- just drop me a message after joining on Patreon.** Also feel free to share you thoughts in the comments, regarding tiers, or anything. 
+
+Update: for users unable to use Patreon there is a [Boosty page](https://boosty.to/hap_player) now.
```
Consider removing the "Update" because it is anyway the first post now?!

baxtan committed:
January 2023

The VL.Addons; It came in a few hours ago, and I included it. Please double-check.

baxtan committed:
Gallery Enteries

Project Dinosauria University from Lunachmeat is added
